lavamoat/browserify/mmi/policy.json

```diff
@@ -1255,6 +1268,18 @@
         "@metamask/eth-sig-util>tweetnacl": true
       }
     },
+    "@metamask/keyring-controller>@metamask/eth-hd-keyring>@metamask/eth-sig-util": {
+      "packages": {
+        "@ethereumjs/tx>@ethereumjs/rlp": true,
+        "@ethereumjs/tx>@ethereumjs/util": true,
+        "@metamask/keyring-controller>@metamask/eth-hd-keyring>@metamask/eth-sig-util>@metamask/abi-utils": true,
+        "@metamask/utils": true,
+        "@metamask/utils>@scure/base": true,
+        "browserify>buffer": true,
```

Comment:
@MetaMask/policy-reviewers, can you take a look on this package? I don't see it here: https://github.com/MetaMask/eth-sig-util/blob/v8.2.0/package.json

Is it normal?